nexus-sdk-js/src/File/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { FileResponse } from './types';
+import { httpPostFile } from '../utils/http';
+import File from '.';
+
+export async function createFile(
```

Comment:
Should this be `uploadFile` instead?

Reply:
I'm copying the verbiage in the documentation: [create a file](https://bluebrain.github.io/nexus/docs/api/kg/kg-files-api.html#create-a-file-using-post)

Although it can be tricky for getting a file... should there be one method to get a file's metadata and another separate method to get the file's